app/src/main/java/org/piwigo/io/repository/UserRepository.java

```diff
@@ -43,11 +46,19 @@
         String url = userManager.getSiteUrl(account);
         String username = userManager.getUsername(account);
         String password = userManager.getPassword(account);
-        Observable<LoginResponse> result = login(url, username, password);
+        Observable<LoginResponse> result = null;
+        try {
+            result = login(url, username, password);
+        } catch (ExecutionException e) {
+            e.printStackTrace();
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
```

Comment:
and yet again: we need to tell the user more about what was going wrong. And if possible, we should even allow to transmit the exception as bug report via ACRA, in case the user doesn't understand what is going wrong. -> with this we can improve the error description if we see more common mistakes.